README.md

```diff
@@ -41,12 +41,23 @@ conan remote add ess-dmsc https://artifactory.esss.lu.se/artifactory/api/conan/e
 
 The ESS remote is required for QtColorWidgets. In case of changes to their deployment infrastructure, check for updates [here](https://github.com/ess-dmsc/conan-configuration).
 
-Clone code and build:
+Clone the code, install dependencies with conan
 ```bash
 git clone https://github.com/martukas/nuclei.git
-mkdir ./nuclei/build
-cd nuclei/build
-cmake ..
+cd nuclei
+conan install . --output-dir=build --build=missing
+```
+Configure and build
+
+If your version of cmake is 3.23 or newer
+```bash
+cmake --preset conan-release
+cmake --build --preset conan-release
+```
+If your version of cmake is older than 3.23
+```bash
+cd build
+cmake .. -DCMAKE_TOOLCHAIN_FILE=conan_toolchain.cmake -DCMAKE_POLICY_DEFAULT_CMP0091=NEW -DCMAKE_BUILD_TYPE=Release
```

Comment:
Wow, yeah this does look way more complicated than I'm used to. I understand "Release", but can't the other things just already be defined somewhere in the CMake files?
And this CMP0091 thing also looks like a sign of something funky. Does this have to do anything with accommodating old ABIs? I think the more common approach is to run `conan profile update settings.compiler.libcxx=libstdc++11 default`  after the conan config detect.

Reply:
I agree, I would suggest putting inside CMakeLists : if cmake version... Then set policy ..

And maybe also:
```
set(default_build_type "Release")
if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
  message(STATUS "Setting build type to '${default_build_type}' as none was specified.")
  set(CMAKE_BUILD_TYPE "${default_build_type}" CACHE STRING "Choose the type of build." FORCE)
  # Set the possible values of build type for cmake-gui
  set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release")# "MinSizeRel" "RelWithDebInfo"
endif()
```

Reply:
>  I think the more common approach is to run `conan profile update settings.compiler.libcxx=libstdc++11 default` after the conan config detect.

This functionality was [removed](https://github.com/conan-io/conan/issues/12833) in v2

I copied the advice/instruction given by conan once it was run, but I think there like to be explicit, so we can safely drop 2 of the command line options as:
- The build type is checked and set in [EnsureBuildType.cmake](https://github.com/martukas/nuclei/blob/main/cmake/EnsureBuildType.cmake) so a new user doesn't need to explicitly set it.
- The `CMP0091` setting is [MSVC](https://cmake.org/cmake/help/latest/policy/CMP0091.html) specific. Looks like this project only checks for MSVC in the boost config and not in a way related to this setting.

The other option (`CMAKE_TOOLCHAIN_FILE`) tells cmake where the packages are installed so is definitely needs. We could assume the user has copy/pasted the command(s), so we know the paths and can set the option in the top level cmake file.

Another option could be to run conan from inside cmake with [execute_process](https://cmake.org/cmake/help/v3.22/command/execute_process.html), but this feels like we are re-inventing what [cmake-conan](https://github.com/conan-io/cmake-conan) is doing so there must be some intricacies I'm not aware of.